ppr-ui/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ppr-ui",
-  "version": "0.3.78",
+  "version": "0.4.0",
```

Comment:
With the upcoming sbc pilot and eventual release of Search, thought it was time to move up.

Reply:
Good, I think this was a long overdue.